cmd/lotus-fountain/site/datacap.html

```diff
@@ -1,15 +1,44 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
     <title>Grant DataCap - Lotus Fountain</title>
     <link rel="stylesheet" type="text/css" href="faucet-components.css">
+    <meta charset="UTF-8">
 	<script src="https://www.google.com/recaptcha/api.js"></script>
 	<script>
    		function onSubmit(token) {
      	document.getElementById("datacap-form").submit();
    	}
 	</script>
-
+    <script type="application/ld+json">
+        [
+            {
+                "@context": "https://schema.org",
+                "@type": "Organization",
+                "name": "Filecoin Lotus Calibration Faucet",
```

Comment:
I suggest naming it 'Filecoin Calibration Faucet' instead of including 'Lotus' in the name. As the 'Lotus' doesn't contribute to the overall clarity.

Reply:
Yes, I think that makes sense

Reply:
Ok. I've made updates 🙏